corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -761,6 +772,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         PersistentCaseTileView: function (options) {
             return new PersistentCaseTileView(options);
         },
+        updateContinueButtonText: updateContinueButtonText,
```

Comment:
Only for "public" methods that get used in other js modules/file need to be added here.